pages/index.md

```diff
@@ -213,30 +213,27 @@ wentout/mnemonica
     were brave enough to share your creations with us. Therefor this is the most beautiful part of 
```

Comment:
"therefore"

Reply:
You're right, thanks a lot for this correction. Fixed